deploy/manager_dev.yaml

```diff
@@ -0,0 +1,46 @@
+# Scylla Manager image
```

Comment:
I'd keep the heml vars somewhere else so it doesn't confuse users under deploy/ dir which isn't really connected to helm. We may change the source in the future, split it, or not.

Reply:
moved to helm/deploy/